blivet/platform.py

```diff
@@ -468,6 +468,8 @@ def get_platform():
             return omapARM()
         else:
             return ARM()
+    elif arch.is_aarch64():
+        return ARM()
```

Comment:
Please create a new ``Aarch64NonEFI`` (or a better name) subclass of the ``ARM`` class and use that here. And don't forget to add a descriptive docstring to that new class explaining why it's just a subclass of ``ARM``.